react-native/src/api/contact/methods.js

```diff
@@ -0,0 +1,28 @@
+import { Platform } from 'react-native-web'
+
+import { CONTACT } from './mutations'
+import ApiClient from '../client'
+import { SERVER_URL } from '../../lib/enums'
+import config from '../../config'
+
+const { environment } = config
+const { currentInstance } = ApiClient
+
+export const sendContactMessage = async contactState => {
+  try {
+    const { data } = await currentInstance().mutate({
+      mutation: CONTACT,
```

Comment:
Change the mutation name to `CREATE_APP_REVIEW` and the method name to `createAppReview`